config.json

```diff
@@ -259,7 +259,11 @@
     {
       "difficulty": 1,
       "slug": "luhn",
-      "topics": []
+      "topics": [
+	"Algorithms",
+	"Integers",
+	"Strings"
```

Comment:
I'd push it up to a 2 at least, if not a 3. While the solution can be fairly step-by-step, there's still a fair amount of steps to consider to build up the necessary algorithm. This is a lot more difficult than hello-world, and not something I think would be given to a new programmer alongside hello-world, I'd be concerned if this exercise appeared earlier in the track, being considered almost as easy as hello-world, as it has a lot to wrap your head around. Also, I noticed you requested help for that exercise? Was that because you were stuck? If so, I think that would indicate the difficulty should be higher than a 1 perhaps.